### PR TITLE
[bgpcfgd] bgp peer is not added successfully sometimes

### PR DESCRIPTION
**Pull request by tylerlinp**
**- What I did**
Fix the issue https://github.com/Azure/sonic-buildimage/issues/3862 (https://github.com/Azure/sonic-utilities/issues/782).

**- How I did it**
Checking meta existance first in peer handler, if not then return false.

**- How to verify it**
Test many times by rebooting with bgp configs(with vrf or not).

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by tylerlinp** [comment]
retest vs please

File: dockers/docker-fpm-frr/bgpcfgd [hunk 1]
```diff
@@ -283,6 +283,8 @@ class BGPPeerMgr(Manager):
         }
 
     def set_handler(self, key, data):
+        if not self.directory.path_exist("meta", "localhost/bgp_asn"):
+            return False
```
Comment (pavel-shirshov) [inline dockers/docker-fpm-frr/bgpcfgd]: I think that the change must be following:
```
     def on_deps_change(self):
+        if not self.directory.available_deps(self.deps):
+            return
         new_queue = []
         for key, data in self.set_queue:
             res = self.set_handler(key, data)
```
I'll have it on my new PR I'm currently working.

**Comment by pavel-shirshov** [comment]
I'd suggest to close your PR and merge https://github.com/Azure/sonic-buildimage/pull/4027

File: dockers/docker-fpm-frr/bgpcfgd (hunk 1 above)
Comment (tylerlinp) [reply dockers/docker-fpm-frr/bgpcfgd]: > I'd suggest to close your PR and merge #4027

OK, thanks!

**Pull request closed without merging**